docs/index.rst

```diff
@@ -28,6 +28,11 @@ Get started with a hands-on introduction for beginners.
 
 Guides and recipes for common problems and tasks, including how to contribute.
 
+:ref:`Topic guides <topic>`
+---------------------------
+
+Thorough explanations of a single topic.
+
 :ref:`Background <background>`
 ------------------------------
```

Comment:
I've just realised that this `background` link points to "about Briefcase"... so there's probably some cleanup there as well...

Reply:
@freakboy3742 Ok I can't figure out what's going on with https://github.com/beeware/briefcase/pull/2335/commits/48e776b342c88ba7e384a3e7dd1b9e5d176e63de. I renamed `background` to `about` in all the places I can find, however, the About section only shows up in the sidebar when you are on the index of the documentation. As soon as you click into a section, including About, it disappears. I'm clearly missing some Sphinx file somewhere, but I can't find what I'm expecting to find. Assistance is appreciated.